src/Menu/Menu.component.tsx

```diff
@@ -3,119 +3,147 @@ import * as React from 'react';
 // VENDOR
 import classNames from 'classnames';
 import styled, { css } from '@xstyled/styled-components';
-import { th } from '@xstyled/system';
+import { th, variant } from '@xstyled/system';
 import { lighten } from 'polished';
-// COMPONENTS
+// ANCHOR
 import { ItemProps } from './Item/Item.component';
+import { space as spaceStyles, SpaceProps } from '@xstyled/system';
 
-interface MenuProps extends React.HTMLAttributes<HTMLDivElement> {
+enum MenuSizes {
+    sm = 'sm',
+    md = 'md',
+    lg = 'lg',
+}
+
+export type MenuSizeProps = MenuSizes | 'sm' | 'md' | 'lg';
+
+interface MenuProps extends React.HTMLAttributes<HTMLDivElement>, SpaceProps {
     children?:
         | Array<React.ReactElement<ItemProps>>
         | React.ReactElement<ItemProps>;
     className?: string;
-    size?: 'small' | 'large';
+    size?: MenuSizeProps;
     // Color
     background?: string;
     color?: string;
     // Configuration
     justify?: 'flex-start' | 'flex-end';
 }
 
+const MenuSizeStyles = {
+    sm: {
+        menu: css``,
+        items: css`
+            font-size: 0.75rem;
+            line-height: 0.75rem;
+            font-weight: 500;
+            padding: 0.5rem 0.75rem;
+        `,
+    },
+    md: {
+        menu: css``,
+        items: css`
+            border-radius: base;
+            font-weight: 600;
+            padding: 1rem 0.75rem;
+        `,
+    },
+    lg: {
+        menu: css``,
+        items: css`
+            border-radius: base;
+            font-weight: 600;
+            padding: 1.5rem 1.25rem;
+        `,
+    },
+};
+
+const sizeVariant = variant({
+    key: 'menu.sizes',
+    prop: 'size',
+    default: 'md',
+    variants: MenuSizeStyles,
+});
+
 const StyledMenu = styled('nav')<MenuProps>`
-  margin: 0;
-  padding: 0;
-  display: flex;
-  width: 100%;
-  min-width: 250px;
-
-  ${({ background = 'primary.base' }) =>
-      css({
-          background: th.color(background),
-      })};
-  font-size: 0.875rem;
-  font-family: ${th('typography.fontFamily')};
-  border-radius: base;
-
-  /* TODO: use React.clone to append a run-time class here instead of using a * selector */
-  > * > a,
-  > a {
-    display: inline-block;
-    line-height: 1.125rem;
-    ${({ color = 'neutrals.white.base' }) =>
+    display: flex;
+    width: 100%;
+    ${spaceStyles};
```

Comment:
This is probably my bad, but 
```
${spaceStyles};
```
should go after
```
margin: 0;
padding: 0
```

so that it doesn't get overritten